PROJECTS.md

```diff
@@ -0,0 +1,47 @@
+# Projects
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Project Facts](#project-facts)
+
+## Overview
+
+Eiffel Community is home to various projects that enable the members of the community to use, contribute,
+and collaborate on Eiffel.
+
+The projects that are part of Eiffel Community adhere to the processes and procedured set by the community
+and follow the [project lifecycle process](./PROJECT_LIFECYCLE.md) for their creation, progression, and
+archival.
+
+Projects hosted in Eiffel Community could utilize one or more repositories under Eiffel Community organization
+on GitHub and the main project repository is where the project facts are documented.
+
+This document lists existing projects, links to the main project repository, and the lifecycle stage the
+projects currently reside in.
+
+Any change to the project facts listed in this document shall be done according to the instructions documented
+in [project lifecycle process](./PROJECT_LIFECYCLE.md).
+
+## Project Facts
+
+The table below lists the projects hosted by Eiffel Community alphabetically. (except the protocol itself)
+All the projects are placed in to [Sandbox](./PROJECT_LIFECYCLE.md#stage-sandbox) stage by default.
+
+Please see [project lifecycle process](./PROJECT_LIFECYCLE.md) for the project lifecycle and the stages.
+
+| Project Name            | Project Description                                             | Project Main Repository                                             | Project Lifecycle Stage                         |
+| ----------------------- | --------------------------------------------------------------- | ------------------------------------------------------------------- | ----------------------------------------------- |
+| Eiffel Protocol         | TBD                                                             | [Link](https://github.com/eiffel-community/eiffel)                  | [Sandbox](./PROJECT_LIFECYCLE.md#stage-sandbox) |
+| Eiffel Commons          | TBD                                                             | [Link](https://github.com/eiffel-community/eiffel-commons)          | [Sandbox](./PROJECT_LIFECYCLE.md#stage-sandbox) |
+| Eiffel Easy2Use         | TBD                                                             | [Link](https://github.com/eiffel-community/eiffel-easy2use)         | [Sandbox](./PROJECT_LIFECYCLE.md#stage-sandbox) |
+| Eiffel Event Repository | TBD                                                             | [Link](https://github.com/eiffel-community/eiffel-event-repository) | [Sandbox](./PROJECT_LIFECYCLE.md#stage-sandbox) |
+| Eiffel Gerrit Herald    | TBD                                                             | [Link](https://github.com/eiffel-community/eiffel-gerrit-herald)    | [Sandbox](./PROJECT_LIFECYCLE.md#stage-sandbox) |
```

Comment:
Was there a reason not to include:

- eiffel-gerrit-plugin https://github.com/eiffel-community/eiffel-gerrit-plugin
- eiffel-jira-plugin https://github.com/eiffel-community/eiffel-jira-plugin

Reply:
nope. i probably missed while browsing the repositories in GitHub Eiffel Community organization. Thanks for the pointer, will fix.